addons/hr_holidays/models/resource.py

```diff
@@ -157,6 +160,57 @@ def unlink(self):
 
         return res
 
+    @api.model
+    def load_public_holidays(self, companies=[], from_ui=False):
+        company_ids = self.env['res.company'].browse(companies)
+        if not company_ids:
+            return
+
+        current_year = datetime.now().year
+        existing_holidays = self.env["resource.calendar.leaves"].search([
+            ('company_id', '=', company_ids.ids),
```

Comment:
It should be 'in', + you already have an id list as a variable :sweat_smile: 
```suggestion
            ('company_id', 'in', company_ids),
```